composer.json

```diff
@@ -18,12 +18,17 @@
         "symfony/console": "^3.2"
     },
     "require-dev": {
-        "phpunit/phpunit": "*"
+        "phpunit/phpunit": "*",
+        "squizlabs/php_codesniffer": "^2.8"
```

Comment:
@sandermangel Sure have https://github.com/phpro/grumphp/commits?author=adam-paterson.

Once I've finished my docker build for the project I'll look at a PR for adding Grump